getting-started/kubernetes/vpp/vm.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Test VPP on a VM-based cluster
```

Comment:
I would be a bit reluctant to remove this one, because even if it is also our dev setup, it is the simplest way to deploy a cluster with the VPP dataplane (there is nothing to configure). Several users went that route to try Calico/VPP.

Reply:
Makes sense :)